jenkins/release-workflows/release-manifest-commit-lock.jenkinsfile

```diff
@@ -155,7 +155,8 @@ pipeline {
                         selectedComponents.each { componentName ->
                             def existingComponent = existingManifest.components.find { it.name == componentName.name }
                             if (existingComponent) {
-                                def releaseTag = params.RELEASE_VERSION + ".0"
+                                def versionTokenize = params.RELEASE_VERSION.tokenize('-')
+                                def releaseTag = versionTokenize[0] + ".0" + "-${versionTokenize[1]}"
```

Comment:
The problem is, if there is no qualifier it will end up with `3.0.0.0-`.
Suggest check if `versionTokenize[1]` is empty before adding.

Thanks.

Reply:
Handled in recent commit

Reply:
Or we could follow how we did in tag creation process. 
```
if (version.contains("-")) {
  tagVersion = version.split("-").first() + ".0-" + version.split("-").last()
} else {
  tagVersion = "$version.0"
}
```
https://github.com/opensearch-project/opensearch-build-libraries/blob/fdfcd30092b6c730a33ac9d36265ad2cccaed5d8/vars/createReleaseTag.groovy#L31-L35
Anyway would work.